src/v2/Apps/Consign/Routes/NewMarketingLanding/Components/__tests__/FAQ.jest.tsx

```diff
@@ -0,0 +1,58 @@
+import { FAQ } from "../FAQ"
+import { ContextModule, OwnerType } from "@artsy/cohesion"
+import { fireEvent, render, screen } from "@testing-library/react"
+import { useTracking, AnalyticsSchema } from "v2/System/Analytics"
+
+jest.mock("v2/System/Analytics/useTracking")
+
+jest.mock("v2/System/Router/useRouter", () => ({
+  useRouter: jest.fn(() => ({
+    match: { params: { id: "1" } },
+  })),
+}))
+
+const trackEvent = useTracking as jest.Mock
+
+describe("FAQ", () => {
+  beforeEach(() => {
+    ;(useTracking as jest.Mock).mockImplementation(() => {
+      return {
+        trackEvent,
+      }
+    })
+  })
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  it("toggles FAQ questions", () => {
+    render(<FAQ />)
+
+    const question = screen.getByText("What does it cost to sell with Artsy?")
+    expect(question).toBeInTheDocument()
+    expect(
+      screen.queryByText(/^There are no upfront fees./)
+    ).not.toBeInTheDocument()
+
+    fireEvent.click(question)
+
+    expect(
+      screen.queryByText(/^There are no upfront fees./)
+    ).toBeInTheDocument()
+  })
+
+  it("tracks a FAQ click with correct params when submission id found", () => {
```

Comment:
**issue:** The description of this test doesn't match the test.

Reply:
Looks correct. Why do you think it's wrong?

Reply:
The test doesn't assert anything about a submission ID being found.

Reply:
Updated test to make it easy to understand. The submission id comes from the router, but the configuration for the test was at the very beginning